### ISSUE TITLE
Lightsaber loading bar option

### ISSUE THREAD

**Issue by mhmtszr** [christopherosthues/starwars-progress-bar#218]
Wouldn't it be great to have a progress bar with a lightsaber effect like Spotify does?

![image](https://github.com/user-attachments/assets/fd14b904-e33a-43cf-9de3-2e7556354a52)

**Comment by christopherosthues** [comment]
Yeah. I'll think about it. But I would have to make a lot of changes to the code base.

**Comment by christopherosthues** [comment]
Hey! 🎉 Just wanted to let you know that the lightsaber-style progress bar enhancement has been implemented.
